rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-configuring.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-nodes-machinepools-configuring"]
+= Configuring machine pools
+:context: rosa-nodes-machinepools-configuring
+
+toc::[]
+Configuration of machine pools.
```

Comment:
I think this line can be removed since it's already covered by the title (I know it's guidelines, but I think it'll still work): Configuration of machine pools.